src/index.js

```diff
@@ -96,13 +96,20 @@ const getCheckedRadio = function (nodes, form) {
   }
 };
 
+const escapeCSS =
+  typeof CSS === 'undefined' || typeof CSS.escape !== 'function'
+    ? function (string) {
+        return string;
```

Comment:
Let's at least plug the injection hole either way by doing `return string.replace(/"/g, '\\"')` and add a test for both the case where `CSS` is defined and where it isn't, if possible.

Reply:
I'll add a replace, but for mitigating injections specifically, I think `\` also needs added for the characters to replace (As a raw name of `\"` would still defeat it).  There may also be ways to defeat that too, my experience with these sort of issues has always been to defer to library given how notorious they are to handle incorrectly yourself.  Though, as noted in the issue, the consequences for getting it wrong or not handling it 100% are pretty minor compared to most injection scenarios.  

I'll add a commit to replace `"`, and then defer to what you think is reasonable in terms of the other special characters to cover for this partial polyfill.

Reply:
This is true. I fiddled a little more with it and it's a downward slope toward, well, `CSS.escape`. And reflecting a little more on it, the escape is still within the realms of `querySelectorAll()`, which should itself protect against worse things than getting an invalid selector, or one that causes other elements to be selected.

I guess I'm more convinced now to just to nothing and return whatever `string` is if there's no`CSS.escape` available (what you had in the first place). For IE users (hopefully very _few_ now...) that also don't have the polyfill, they'll get the same behavior as today (error), but now they'll have the option to polyfill and we'll support it.

Thanks for being willing to add (and adding) the check for quotes, but let's revert that.

Reply:
Another option is to provide a safe 'quarySelectorAll' similar to 'matchas' in order to catch the possible error in browsers without 'CCS.escape'

Reply:
Yes, I thought of that too, but then I got thinking that if we did that, or just wrapped in try..catch here, it would fail silently and be worse than the exception being thrown since users wouldn't know why the radios aren't being found. I guess we could try..catch and print a more informative error to the console in that case to be more helpful.

Reply:
I think that for this specific flow I would rather get the different behavior then an error that might break some higher level functionality

Reply:
So you'd rather we catch the error, maybe `console.error("Looks like you have a radio button with a name attribute containing invalid CSS selector characters and need the CSS.escape polyfill: %s", err.message)` and then skip the radio buttons and continue execution?

Reply:
I've added the `try / catch` and reverted the `replace` in the fallback.  I'd like to add a test but I'm having a hard time figuring out how to fit it in with the existing test structure.  The ways I've thought of to have the fallback be active in the test is either:

1. Add the IE karma plugin and run it (which is a whole can of worms)
1. Remove `escape` from the `CSS` object and re-import the module (hacky)
1. Change the logic to check if a fallback is necessary every time the function runs, instead of when the module is imported, and use mocks or environment variables to force the fallback in tests (not ideal as the implementation has test-specific logic, and it's not obvious what can cause a test failure if the implementation changes)
1. Use a test outside of a browser that can more easily setup mocks to force the fallback (not ideal to have one or two tests break from convention.  Also additional devops overhead and maintenance)

It's also very possible that I'm missing something obvious, so let me know any suggestions or ideas you might have for this @stefcameron @idoros

Reply:
Could you just do something like this?
```javascript
it('should work with fallback', () => {
  const previous = CSS.escape;
  CSS.escape = undefined;

  // test...

  if (previous) {
    CSS.escape = previous;
  }
});
```
Maybe not the most elegant, but it's a pattern that has worked for me in the past in similar situations since `CSS` isn't a module that gets loaded (i.e. it doesn't need to be mocked in some special Karma way. It's just provided by the environment).

Reply:
Hmm, the issue I found with that approach is the `escapeCSS` function is only assigned when the module is imported.  So even if `CSS.escape` is set to undefined, all of the `tabbable` functions will still use the previous, valid, reference to `CSS.escape`.  It's possible to modify it to something like this:

```js
function escapeCSS(string) {
  return typeof window === 'undefined' ||
    typeof window.CSS === 'undefined' ||
    typeof window.CSS.escape !== 'function'
    ? string
    : window.CSS.escape(string);
}
```

but that comes with the drawback that it's very easy to update the function back to a form where the reference to the `escape` function used isn't always using the latest `CSS.escape` value.  If that happens, the test has no way of knowing and will continue to pass even though the fallback will no longer be tested.  There's no easy way with the current setup to detect that lack of coverage and fail.

Reply:
Well, I think I found something that works by writing assertions against the added `console.error` message.  Let me know if you think it looks okay.  Particularly with adding `sinon` as a devDependency to help out with stubbing, since it's pretty heavyweight compared to what the test needs it for.